dogowner/test_dog_owner.py

```diff
@@ -163,3 +163,14 @@ def test_dog_owner_user_creation_with_url_not_an_image(self):
                             dog_picture_url="https://www.not/an/image.com",
                             dog_age=10, dog_weight=6, dog_gender='M'
                             )
+
+    def test_dogowner_has_customized_profile_image(self, create_dog_owner_user):
+        dogowner_profile_image = create_dog_owner_user.get_dog_owner_profile_image_url()
+        assert dogowner_profile_image != "../../static/images/dog-owner-default-profile-picture.jpeg"
+        assert dogowner_profile_image is not None
```

Comment:
this test is redundant if profile_image equals the wanted image, so of course its not None